tox.ini

```diff
@@ -15,6 +15,8 @@ deps =
 commands = nosetests tests
 deps =
     nose
+    futures
```

Comment:
I wonder if migrating to Python 3 should be prioritized, given EOL plans for Py2.

Reply:
yes, that would be good, but not for this PR I guess